### PR TITLE
list pat and authorized oauth apps

### PR DESCRIPTION
**Pull request by tanay-microsoft**
List of PAT tokens and authorized OAuth applications only if principal name is not provided!

### CODE REVIEW

File: src/AzSK.ADO/AzSKADOInfo/AzSKADOInfo.ps1 [hunk 1]
```diff
@@ -163,13 +163,15 @@ function Get-AzSKADOInfo
 							Write-Host "This command currently supports user permission information for a single project. Please provide a single project name." -ForegroundColor Red
 						}
 						elseif ([string]::IsNullOrWhiteSpace($PrincipalName)) {
+							$principal_flag = $true;
 							$currentUser = [ContextHelper]::GetCurrentSessionUser();
 							# Write-Host "InfoType 'UserInfo' requires principal name parameter to scan. If not provided, it'll take the context of current logged in user." -ForegroundColor Yellow
-							$userInfo = [UserInfo]::new($OrganizationName, $currentUser, $ProjectNames, $PSCmdlet.MyInvocation);
+							$userInfo = [UserInfo]::new($OrganizationName, $currentUser, $principal_flag, $ProjectNames, $PSCmdlet.MyInvocation);
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/AzSKADOInfo/AzSKADOInfo.ps1:169]: change all variables to camelcase

File: src/AzSK.ADO/Framework/Core/AzSKADOInfo/UserInfo.ps1 [hunk 2]
```diff
@@ -163,6 +165,72 @@ class UserInfo: CommandBase {
 					# [EventBase]::PublishGenericException($_);
 				}
 			}
+			$this.PublishCustomMessage([Constants]::DoubleDashLine)
+			$returnMsgs += [Constants]::DoubleDashLine;
+			if($this.principal_flag) {
```
Comment (Garima-msft) [inline src/AzSK.ADO/Framework/Core/AzSKADOInfo/UserInfo.ps1:170]: Add comments

File: src/AzSK.ADO/AzSKADOInfo/AzSKADOInfo.ps1 (hunk 1 above)
Comment (v-soukum) [reply src/AzSK.ADO/AzSKADOInfo/AzSKADOInfo.ps1]: Changes done.

File: src/AzSK.ADO/Framework/Core/AzSKADOInfo/UserInfo.ps1 (hunk 2 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Core/AzSKADOInfo/UserInfo.ps1]: Comments added

**Review by v-himkam — APPROVED** [review]

**Review by v-soukum — APPROVED** [review]

**Pull request merged**